cookbooks/aws-parallelcluster-slurm/recipes/config_head_node.rb

```diff
@@ -173,6 +173,22 @@
   action :create
 end
 
+cookbook_file '/opt/slurm/etc/pcluster/prolog' do
+  source 'head_node_slurm/prolog'
+  owner node['cluster']['cluster_admin_user']
+  group node['cluster']['cluster_admin_group']
+  mode '0744'
+  action :create
+end
+
+cookbook_file '/opt/slurm/etc/pcluster/epilog' do
+  source 'head_node_slurm/epilog'
+  owner node['cluster']['cluster_admin_user']
+  group node['cluster']['cluster_admin_group']
+  mode '0744'
+  action :create
+end
```

Comment:
aren't these scripts executed as the slurm user? I'm wondering how Slurm is able to execute them with these permissions

Reply:
Sorry. It might be better to use the slurm user. I set it to admin because I thought the slurm_resume log had the same permission.
But the tests are passed. Probably a read permission for the prolog/epilog is enough

Reply:
Changed to Slurm user